pkg/volume/scaleio/sio_volume_test.go

```diff
@@ -84,7 +84,7 @@ func TestVolumeCanSupport(t *testing.T) {
 	defer os.RemoveAll(tmpDir)
 	plug, err := plugMgr.FindPluginByName(sioPluginName)
 	if err != nil {
-		t.Errorf("Can't find the plugin %s by name", sioPluginName)
+		t.Fatal("Can't find the plugin %s by name", sioPluginName)
```

Comment:
need you use Fatalf?

Reply:
because the following code uses the var `plug`, it might be nil
https://github.com/kubernetes/kubernetes/blob/72e3c551f6d03e4b7b9915c5dc2f23c88e67da33/pkg/volume/scaleio/sio_volume_test.go#L89

Reply:
I mean you either
```
t.Fatalf("Can't find the plugin %s by name", sioPluginName)
```
or 
```
t.Fatal("Can't find the plugin by name")
```

Reply:
sorry, I didn't notice the argument